hybrid/compartment_examples/inter_comp_call/secure/compartments.s

```diff
@@ -30,7 +30,6 @@ comp_f_fn_end:
  */
 .type comp_g_fn, "function"
 comp_g_fn:
-    mrs       c10, DDC
```

Comment:
Is this intentional?

Reply:
Good catch, I actually meant to submit this as a separate PR - it looks like this instruction isn't needed at all (`c10` is never referenced again in `comp_g_fn`).

Reply:
Fixed in [642eac5](https://github.com/capablevms/cheri-examples/pull/75/commits/642eac5dfcc1acad856cf800917446e12cf3cddd)